Resources/Prototypes/Entities/Objects/Materials/materials.yml

```diff
@@ -182,3 +182,27 @@
     state: cotton
   - type: Stack
     count: 1
+
+- type: entity
+  parent: MaterialBase
+  id: FissionFuel
+  name: fission fuel
+  suffix: Full
+  description: Radioactive Fuel made for Nuclear Fission Engines.
+  components:
+  - type: Stack
+    stackType: FissionFuel
+  - type: Sprite
+    state: diamond
+  - type: Item
+    HeldPrefix: diamond
+
+- type: entity
+  parent: FissionFuel
+  id: FissionFuel1
```

Comment:
Use the word one or single or something more verbose

Reply:
That's pretty in-line with other materials. E.g. SheetGlass1, MaterialDurathread1, etc

Reply:
> That's pretty in-line with other materials. E.g. SheetGlass1, MaterialDurathread1, etc

Is it? alright thats fine then